services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -386,7 +441,27 @@ def compute_descriptive_statistics_response(
     con.sql("INSTALL httpfs")
     con.sql("LOAD httpfs")
     con.sql("SET enable_progress_bar=true;")
+    con.sql(f"CREATE TABLE {DATA_TABLE_NAME} as SELECT * FROM read_parquet('{local_parquet_glob_path}');")  # nosec
```

Comment:
this takes more RAM but should make the querying faster

Reply:
OK! I was wondering why using this. Will it break our workers on big datasets (GBs or TBs of parquet files)?

Reply:
Or are the statistics always computed on the first 5GB anyway?

Reply:
(not for this PR): if we limit to the first 5GB, and if we use duckdb, and if we already have a duckdb file for the first 5GB (used for search and filter), why not using this file instead of the parquet files?

Reply:
We only create the duckdb index file if there are string columns

Reply:
We only create the duckdb index file if there are string columns

Reply:
i thought an index file is a file with search index, not with dataset data, no?

Reply:
> i thought an index file is a file with search index, not with dataset data, no?

In our case, it has the data + the index

Reply:
> Or are the statistics always computed on the first 5GB anyway?

ah actually only on the datasets < 5gb in size

Reply:
> In our case, it has the data + the index

ah okay. but only string columns data, right?

Reply:
no, all of them